sections/round_structure.tex

```diff
@@ -26,6 +26,13 @@
 
 \vspace*{\fill}
 
-\includegraphics[width=1.8\linewidth]{\art/minotaur.png}
+\includegraphics[width=1.4\linewidth]{\art/minotaur.png}
+
+\vspace*{\fill}
+
+\note{9}{
+  During the Alliance game, you play in teams, which activate alternately until all players have taken their turn.
```

Comment:
Why note and not regular text?

Reply:
Because that's an unusual rule which doesn't apply to most games, but only in certain conditions?

That's called an exception:

![Выделение_582](https://github.com/Heegu-sama/Homm3BG/assets/325092/a2f7bc5a-e5da-4070-af60-65024be91a05)

And... I've just noticed two links which should probably stay hyperlinks instead of pagelink :D

Reply:
You mean those on the intro page?

Reply:
Yeah, it is quite long

Reply:
Two ideas for shorter versions

> In the Alliance game, teams take turns alternately until all players have played. When it's your team's turn, you can choose who goes first if no one from your team has played that round, allowing you to swap players each turn.

> In the Alliance game, teams take turns alternately. Each round, you can choose who goes first within your team if no one has played yet.

Reply:
+1 for the latter